docs/contributors/code/release.md

```diff
@@ -45,10 +49,14 @@ When editing the notes, you should be sure to:
 
 You can find some more tips on writing the release notes and post in the section below.
 
+#### Publishing the release
+
+**Note**: you may not want to undertake this step until the release post is ready on [make.wordpress.org/core](https://make.wordpress.org/core/) ([see below for more detail](#documenting-the-release-with-the-release-notes-and-post)).
```

Comment:
Why you added that? I think in many cases we release the plugin in order for users to get it on the expected timeline and the release post can be delayed just a bit. I've seen this happening many times.. --cc @priethor

Reply:
I thought I remembered that they had to be released roughly in sync. Yeh maybe we don't need this then. It just caught my eye when I read it.

Reply:
✅